sentry/src/routes.rs

```diff
@@ -59,15 +59,30 @@
 //!
 //! #### GET `/v5/channel/:id/accounting`
 //!
+//! Gets all of the accounting entries for a channel from the database and checks the balances.
+//!
 //! The route is handled by [`channel::get_accounting_for_channel()`].
 //!
-//! Response: [`AccountingResponse::<CheckedState>`](primitives::sentry::AccountingResponse)
+//! Response:
+//!
+//! ```
+#![doc = include_str!("../../primitives/examples/accounting_response.rs")]
+//! ```
 //!
 //! #### GET `/v5/channel/:id/spender/:addr` (auth required)
 //!
+//! Gets the spender limits for a spender on a [`Channel`]. It does so by fetching the
+//! latest Spendable entry from the database (or creating one if it doesn't exist yet) from which
+//! the total deposited amount is retrieved, and the latest NewState from which the total spent
+//! amount is retrieved.
+//!
 //! The route is handled by [`channel::get_spender_limits()`].
 //!
-//! Response: [`SpenderResponse`](primitives::sentry::SpenderResponse)
+//! Response:
```

Comment:
Please keep the "Response: [`SpenderResponse`](primitives::sentry::SpenderResponse)" and include an `Examples` section